tests/grid/__snapshots__/grid_component.test.ts.snap

```diff
@@ -88,7 +88,7 @@ exports[`Grid component simple rendering snapshot 1`] = `
           class="o-composer w-100 text-start"
           contenteditable="true"
           spellcheck="false"
-          style="max-height: 1008.6px;"
+          style="max-height: 982.6px;"
```

Comment:
not sure why the max-height of the inactive grid composer changed with this fix ?